Add example for field search with spaces (#113)
ankitects/anki-manual: src/searching.md

find notes with a Front field of exactly "dog". A field that says "a
dog" will not match.

`"animal front:a dog"`  

> find notes with an Animal Front field of exactly "a dog". The double quotes are

Reviewer: ```suggestion
find notes where the the "Animal Front" field is exactly "a dog". The double quotes are
```
